core/os/device/device.proto

```diff
@@ -155,84 +155,84 @@ message Configuration {
     // The OS the device is running.
     OS OS = 1;
     // The hardware description of this device.
-    Hardware Hardware = 2;
+    Hardware hardware = 2;
     // The abi's the device supports.
     repeated ABI ABIs = 3;
     // The drivers supported by the system.
-    Drivers Drivers = 4;
+    Drivers drivers = 4;
 }
 
 // Instance represents a physical device.
 // An instance is persistable, and can be used to retain information about
 // offline devices, and reconnect to them.
 message Instance {
     // The unique identifier of the instance.
-    ID id = 1;
+    ID ID = 1;
     // The serial code of the device, if present.
-    string Serial = 2;
+    string serial = 2;
     // The friendly name of this device, if present.
-    string Name = 3;
+    string name = 3;
     // The hardware and software configuration of the device.
-    Configuration Configuration = 4;
+    Configuration configuration = 4;
 }
 
 // Drivers describes the drivers available on a device.
 message Drivers {
     // The OpenGL or OpenGL ES driver support.
-    OpenGLDriver OpenGL = 1;
+    OpenGLDriver open_gl = 1;
```

Comment:
nit: I'd probably call this `opengl` but that's just me.

Reply:
done.